lib/manager/git-submodules/readme.md

```diff
@@ -1 +1,4 @@
-Keeps Git submodules updated within a repository.
+Keeps publicly accessible Git submodules updated within a repository.
+
+Renovate does not support updating Git submodules that are hosted on a private repository.
```

Comment:
But that's just not true... It is supported, just maybe requires a bit of preparation for authentication (like `.netrc` token for Github)...

Reply:
Thank you for your idea. I've copied your comment into issue #10149, so that the Renovate team can discuss this idea in the correct place. 😉